tests/integration/tests/test_ingress.py

```diff
@@ -14,23 +15,44 @@
 def test_ingress(session_instance: List[harness.Instance]):
 
     util.stubbornly(retries=5, delay_s=2).on(session_instance).until(
-        lambda p: "cilium-ingress" in p.stdout.decode()
-    ).exec(["k8s", "kubectl", "get", "service", "-n", "kube-system", "-o", "json"])
+        lambda p: "ingress" in p.stdout.decode()
+    ).exec(["k8s", "kubectl", "get", "service", "-A", "-o", "json"])
```

Comment:
same here, we fetch services until we see some arbitrary output, then fetch them again. this should all be done inside the `util.stubbornly` here